Cloud Application Manager/Tutorials/ebcli-tutorial.md

```diff
@@ -0,0 +1,442 @@
+
+**Terminal Command**
+
+```
+ebcli export [--boxes-path <BOXES_PATH>] [-r] ”<box ID>”
+```
+
+
+**Parameters**
+
+| Option | Description |
+|-------------------|-------------|
+| –-boxes-path | Path where the boxes will be exported. |
+| -r | Recursively export box and inner boxes. |
+
+
+## Import a Box
+
+Use the ebcli to import a box from its JSON and text files into Cloud Application Manager or your appliance.
+
+**Terminal Command**
+
+```
+ebcli import [--boxes-path <BOXES_PATH>] [-w <WORKSPACE_ID>] [-m <COMMENT>] [--image <IMAGE>] [--as-draft] box-name
+```
+
+**Parameters**
+
+| Option | Description |
+|-------------------|-------------|
+| –-boxes-path | Path where the boxes are located. |
+| -w | Workspace id where the box will be imported. |
+| –m | Description of the version will be created. i.e. 'Update requirements'. |
+| –-image | Name of the base image to be used. i.e. ubuntu:14.04 or Centos. |
+| –-as-draft | Instead of create a version the box will be imported as draft of the box. |
+
+
+## Deploy an application box
+
+Use the ebcli to deploy an application box in your Cloud Application Manager or your appliance account.
+
+**Terminal Command**
+
+```
+ebcli deploy [-w <WORKSPACE_ID>] [-t [<TAG> [<TAG> ...]]] [-c [<CLAIM> [<CLAIM> ...]]] input
+```
+
+**Parameters**
+
+| Option | Description |
+|-------------------|-------------|
+| -w | Workspace id where the box will be imported |
+| –t | Tag or Tags which will mark and isolate your application instances and bindings. i.e. 'production us-east-1' |
+| –c | Claims to select the correct deployment policies to be used by instances. i.e. 'ubuntu us-east-1' |
+| –input | Application's JSON document to deploy |
+
+
+### Instance lifecycle using ebcli
+
+## Get instances
+
+Use ebcli to list your instances.
+
+**Terminal Command**
+
+```
+ebcli instances [-f [<FIELD> [<FIELD> ...]]] [-t [<TAG> [<TAG> ...]]] [-w “<WORKSPACE ID>”] 
+```
+
+**Parameters**
+
+| Get Boxes Option | Description |
+|-------------------|-------------|
+| -f | Field or Fields you want to be listed. |
+| –t | Tag or Tags used to filter your instances. i.e. 'production us-east-1' |
+| –w | Workspace which contains the instances you want to list. |
+
+
+## Reconfigure an Instance
+
+Use ebcli to execute reconfigure operation on an instance which is powered on.
+
+**Terminal Command**
+
+```
+ebcli reconfigure ”<instance-id>”
+```
+
+**Parameters**
+
+| Option | Description |
+|-------------------|-------------|
+| instance-id | ID of the Instance which boxes you want to reconfigure. |
+
+
+## Reinstall an Instance
+
+Use ebcli to execute reinstall operation on an instance which is powered on.
+
+**Terminal Command**
+
+```
+ebcli reinstall ”<instance-id>”
+```
+
+**Parameters**
+
+| Option | Description |
+|-------------------|-------------|
+| instance-id | ID of Instance you want to power on. |
+
+
+## Terminate an Instance
+
+Use ebcli to execute terminate operation on an instance which is provisioned.
```

Comment:
I would add a similar warning to the one we added in the UI, to alert that this is a destructive operation and the underlying infrastructure will be removed. (you are mentioning it somehow just below, but it's best to make a explicit mention)

Reply:
done